src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -319,4 +320,33 @@ export default class GeBatchGiftEntryTable extends LightningElement {
         }));
     }
 
+    handleMenuItemSelect(event) {
+        if (event.detail.value === 'selectcolumns') {
+            const selectColumns = new CustomEvent('selectcolumns', {
+                detail: {
+                    options: this.getAllColumns()
+                        .map(({label, fieldName}) => ({
+                            label, value: fieldName
+                        })),
+                    values: this.computedColumns
+                        .map(({fieldName}) => fieldName)
+                }
+            });
+            this.dispatchEvent(selectColumns);
+        }
+    }
+
+    get tableMenuQaLocator() {
```

Comment:
Could we rename this with `qaLocator` at the front to align with other components?
```suggestion
    get qaLocatorTableMenu() {
```

Reply:
👍🏽